spec/controllers/jobs_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe JobsController, :type => :controller do
+
+  describe 'GET retry' do
+    it 'returns 404 if there is no id passed' do
+      get :retry
+      expect(response).to have_http_status(404)
+    end
+    it 'retries a job from delayed job queue' do
+      job = Delayed::Job.create({:handler=>''})
```

Comment:
There seems to be a lot of database creating/cleaning going on in the tests. This project may benefit from using something like [FactoryGirl](https://github.com/thoughtbot/factory_girl) and [DatabaseCleaner](https://github.com/DatabaseCleaner/database_cleaner)